pom.xml

```diff
@@ -176,15 +188,6 @@
     </modules>
 
     <profiles>
-        <profile>
```

Comment:
Do yoy think that we must delete this profile (jdk8) or it's a mistake? The require version for PF4J is java 7.

Reply:
Pippo requires 1.8+: `<java.version>1.8</java.version>` The profile was always active (actually should be active but it was not on 9), and the parameter is explicitly set in plugin configuration

Reply:
My mistake, sorry. I am working on a feature on PF4J (a project initiated by me that requires Java 1.7) and from this reason I mix things.